lib/index.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * bwallet entrypoint
+ *
+ */
+
+import MainContainer from './Containers';
+
+export { mapComponentState, mapComponentDispatch, getRouteProps } from './mappings';
+export { reduceNode, reduceChain } from './reducers';
+
+export const metadata = {
+  name: 'wallet',
```

Comment:
do you think it's confusing having the name be `wallet` but the namespace be `bwallet`?

Reply:
Ah good catch

Reply:
in which case we'd want to change the name of the npm package and repo too right?